examples/MODULE.bazel

```diff
@@ -0,0 +1,51 @@
+module(name = "rules_ros_examples")
+
+bazel_dep(name = "rules_python", version = "0.34.0")
```

Comment:
If you now specify your own python toolchain, wouldn't it be possible to use a different python version?

Reply:
It won't let me reply to the above `.bazelrc` question but basically because no toolchain is specified as default in the example MODULE.bazel file, you have to specify it as a config flag in the `.bazelrc`

Reply:
Would it help if we specify a toolchain here?

Reply:
If I add the following to examples/MODULE.bazel:

```Python
```
python = use_extension("@rules_python//python/extensions:python.bzl", "python")
python.toolchain(
    is_default = True,
    python_version = "3.10",
)

and remove the config flag from .bazelrc: that works for the examples repo but I get:

```
DEBUG: /home/madwolf/.cache/bazel/_bazel_user/6e0e647a7496b7382203f7000f7beb27/external/rules_python~/python/private/python.bzl:46:10: WARNING: Ignoring toolchain 'python_3_10' from module 'rules_ros': Toolchain 'python_3_10' from module 'rules_ros_examples' already registered Python version 3.10 and has precedence
```

IMO, the flag in .bazelrc is a cleaner solution.

Reply:
But isn't this what we want? Being able to override the Python toolchain used?

Reply:
Didn't have time to investigate how this would work. If you have some time, give it a shot. But, yes, it would be nice if toolchain override could work.

Reply:
Ok fine with me. Let's try that and once it is merged I will try to update in our codebase and report back if there are any errors.